public/data/guatemalaPage.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
You could have put these location json files into another table in your postgres DB, and used an ajax call to get the data. It would reduce the static resources that you're requesting via HTTP.